apiserver/errors/errors.go

```diff
@@ -218,14 +215,8 @@ func ServerError(err error) *params.Error {
 	case errors.Is(err, params.UpgradeInProgressError),
 		errors.Is(err, upgrade.ErrUpgradeInProgress):
 		code = params.CodeUpgradeInProgress
-	case errors.Is(err, stateerrors.HasAttachmentsError):
-		code = params.CodeMachineHasAttachedStorage
-	case errors.Is(err, stateerrors.HasContainersError):
-		code = params.CodeMachineHasContainers
 	case errors.Is(err, stateerrors.StorageAttachedError):
 		code = params.CodeStorageAttached
-	case errors.Is(err, stateerrors.IsControllerMemberError):
-		code = params.CodeTryAgain
```

Comment:
@jack-w-shaw this needs wiring up during machine removal. i.e. you're not allowed to remove a machine if it's a controller without using force.

Can be done in a subsequent PR.